dry-rails.gemspec

```diff
@@ -29,9 +29,9 @@ Gem::Specification.new do |spec|
   spec.required_ruby_version = ">= 2.6.0"
 
   # to update dependencies edit project.yml
-  spec.add_runtime_dependency "dry-schema", "~> 1.5"
-  spec.add_runtime_dependency "dry-system", "~> 0.18.0", ">= 0.18.1"
-  spec.add_runtime_dependency "dry-validation", "~> 1.5"
+  spec.add_runtime_dependency "dry-schema", "~> 1.8"
+  spec.add_runtime_dependency "dry-system", "~> 0.20.0", ">= 0.18.1"
+  spec.add_runtime_dependency "dry-validation", "~> 1.7"
```

Comment:
I changed `project.yml` but it had no effect, so I bumped it here to enable the work
Please let me know how to update `project.yml` properly so bundler would pick it up 🙏

Reply:
@zlw this is something that only works in master, don't worry about it